### ISSUE TITLE
Add skip to main content-link

### ISSUE THREAD

**Issue by eevajonnapanula** [turkupy/turkupy.github.io#47]
We need a "Skip to main content"-link (especially after #46). You can find [reasoning and how to accomplish this from here](https://css-tricks.com/how-to-create-a-skip-to-content-link/) 

Things to note:
- When not active, that link should be hidden. 
- When active (focused) it should be visible. 
- Add texts as translations as [instructed here](https://github.com/turkupy/turkupy.github.io/blob/dev/docs/translations.md#translations) (So not as hard coded strings)
